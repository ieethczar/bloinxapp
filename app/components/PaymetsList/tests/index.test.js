import React from 'react';
import renderer from 'react-test-renderer';

import PaymentsList from '../index';

describe('PaymentsList', () => {
  const data = [
    {
      id: 'some-id',
      title: 'some-title',
      amount: 10,
      currency: 'some-currency',
      status: 'some-status',
      date: '2020-12-12',
    }
  ];

  it('should render correctly', () => {
    const component = renderer.create(
      <PaymentsList data={data} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});