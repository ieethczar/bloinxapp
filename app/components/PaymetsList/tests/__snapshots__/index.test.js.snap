// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentsList should render correctly 1`] = `
<RCTScrollView
  data={
    Array [
      Object {
        "amount": 10,
        "currency": "some-currency",
        "date": "2020-12-12",
        "id": "some-id",
        "status": "some-status",
        "title": "some-title",
      },
    ]
  }
  disableVirtualization={false}
  getItem={[Function]}
  getItemCount={[Function]}
  horizontal={false}
  initialNumToRender={0}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  onContentSizeChange={[Function]}
  onEndReachedThreshold={2}
  onLayout={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={Array []}
  style={
    Object {
      "height": "100%",
    }
  }
  updateCellsBatchingPeriod={50}
  windowSize={21}
>
  <View>
    <View
      onLayout={[Function]}
      style={null}
    >
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": 0.5,
              "borderColor": "#bcbbc1",
              "flexDirection": "row",
              "padding": 5,
            }
          }
          testID="padView"
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "flex-start",
                  "flex": 1,
                  "justifyContent": "center",
                },
                Object {
                  "display": "flex",
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                },
              ]
            }
            theme={
              Object {
                "colors": Object {
                  "black": "#242424",
                  "disabled": "hsl(208, 8%, 90%)",
                  "divider": "#bcbbc1",
                  "error": "#ff190c",
                  "grey0": "#393e42",
                  "grey1": "#43484d",
                  "grey2": "#5e6977",
                  "grey3": "#86939e",
                  "grey4": "#bdc6cf",
                  "grey5": "#e1e8ee",
                  "greyOutline": "#bbb",
                  "platform": Object {
                    "android": Object {
                      "error": "#f44336",
                      "grey": "rgba(0, 0, 0, 0.54)",
                      "primary": "#2196f3",
                      "searchBg": "#dcdce1",
                      "secondary": "#9C27B0",
                      "success": "#4caf50",
                      "warning": "#ffeb3b",
                    },
                    "default": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                    "ios": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                    "web": Object {
                      "error": "#ff190c",
                      "grey": "#393e42",
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#ca71eb",
                      "success": "#52c41a",
                      "warning": "#faad14",
                    },
                  },
                  "primary": "#2089dc",
                  "searchBg": "#303337",
                  "secondary": "#ca71eb",
                  "success": "#52c41a",
                  "warning": "#faad14",
                  "white": "#ffffff",
                },
              }
            }
          >
            <View>
              <Text
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#242424",
                    "fontSize": 17,
                  }
                }
                testID="listItemTitle"
              >
                Movimiento desconocido
              </Text>
              <Text
                style={
                  Object {
                    "color": "#555555",
                    "fontSize": 12,
                  }
                }
              >
                11 Dec
              </Text>
            </View>
            <View>
              <Text
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#242424",
                    "fontSize": 17,
                  }
                }
                testID="listItemTitle"
              >
                $
                10.00
                 
                some-currency
              </Text>
              <Text
                style={
                  Object {
                    "color": "#555555",
                    "fontSize": 12,
                  }
                }
              >
                some-status
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
